src/app/setup-wizard/pre-visit/pre-assessment-setup/pre-assessment-setup.component.html

```diff
@@ -126,6 +126,36 @@ <h6>Utility Types</h6>
                                                                     </div>
                                                                 </div>
                                                             </ng-container>
+                                                            <ng-container *ngIf="use.include
+                                                                && !(use.utilityType | isStandardUnit :use.energyUnit : 'Energy')">
+                                                                <div class="col-4">
+                                                                    <div class="input-group">
```

Comment:
Make sure this label is lined up with the label above it. Right now it's off by a bit